rust/gui-client/src-tauri/src/client.rs

```diff
@@ -129,6 +170,7 @@ fn show_error_dialog(error: &gui::Error) -> Result<()> {
         gui::Error::UserNotInFirezoneGroup => format!("You are not a member of the group `{FIREZONE_GROUP}`. Try `sudo usermod -aG {FIREZONE_GROUP} $USER` and then reboot"),
         gui::Error::Other(error) => error.to_string(),
     };
+    tracing::error!("{}", error_msg);
```

Comment:
I renamed `error_msg` to `user_friendly_error_msg` here to clarify, it's actually a different message that has less detail. But I thought it would be nice to also log exactly what the user sees, in addition to the detailed dev-friendly logs.